src/components/Kurlsh.js

```diff
@@ -240,6 +240,68 @@ class Kurlsh extends React.Component {
     };
   }
 
+  prepareVersions = versions => {
+    // the list of versions returned by the api has, as it first item, a version called
+    // "latest", the item immediately after it is the actual version we consider to be
+    // the latest. we save it here so we can remember where we need to insert it after.
+    let latest = versions[1].version;
+
+
+    // remove the "latest" version from the list and sort the resulting array. we will
+    // re-insert "latest" after sorting the array as it has to be the option just before
+    // the "actual" latest version.
+    const sorted = versions.filter(obj => {
+        return obj.version !== "latest";
+    }).sort(this.compareVersions);
+
+    let result = []
+    for (let i=0; i < sorted.length; i++) {
+      if (sorted[i].version === latest) {
+        result.push({version: "latest"});
+      }
+      result.push(sorted[i]);
+    }
+
+    result.push({ version: "None" });
+    return result;
+  }
+
+  // compareVersions do the best to sort out versions returned by the api. this function
+  // only considers versions that ressembles the semantic version format, if the version
+  // does not look like a semantic version this function simply returns 0, i.e. mantain
+  // the original order.
+  compareVersions = (x, y) => {
+    const version1 = x.version.split('.');
+    const version2 = y.version.split('.');
+    if (version1.length !== 3 || version2.length !== 3) {
```

Comment:
I was trying to avoid adding a new dependency just for that but it seems like this is already a dependency in the project. Thanks for the heads up, fixed.